packages/plugin-html-vas-response/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@jspsych-contrib/plugin-html-vas-response",
+  "private": "true",
+  "version": "1.0.0",
```

Comment:
Can you bump this down to version 0.1 so that the release action will publish version 1.0?

Reply:
And `"private": "true",` needs to be removed too. It's what prevents the template plugins themselves from being published...

Reply:
I've just made these changes!